Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -364,6 +355,9 @@ - (UIBarButtonItem*)shareToolbarItem {
 - (void)viewDidLoad {
     [super viewDidLoad];
 
+    [self setupToolbar];
```

Comment:
@bgerstle FYI - From what I read on the link you posted, Apple does not really recommend setting tool bar items on init in that example, they happen to be walking the reader through some some sample code and only make mention of initialization in a figure:

> Listing 1-3 shows the code needed to specify the toolbar items programmatically. You would implement the method in your view controller and call it at initialization time.

And then go on to say that you are able to change toolbaritems after init:

> In addition to setting toolbar items during initialization, a view controller can also change its existing set of toolbar items dynamically using the setToolbarItems:animated: method…

Which we already know. This points to there being no technical reason to set toolbar items in init. If you know of one let me know. 

However, setting toolbar items in `viewDidLoad` is insanely common - I would venture to guess more common than setting in `init`. If that isn't allowed, then we and a lot of developers have a lot of code to go through and change.